{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/input.jinja

```diff
@@ -13,7 +13,7 @@
             @focus="updateFlag" @blur="updateFlag" @input="updateFlag"
         {% endif %}
         x-data="input('update-{{ event_name | replace('_', '-') }}', '{{ value }}')"
-        x-ref="input_{{ name }}" class="input {{ input_classes }} {{attrs.class}}" {% do attrs.update({"type": type, "name": name, "value": value}) %} {{ attributes(attrs)}} />
+        x-ref="input" class="input {{ input_classes }} {{attrs.class}}" {% do attrs.update({"type": type, "name": name, "value": value}) %} {{ attributes(attrs)}} />
```

Comment:
I think this breaks if you have mutlple imputs in a form

Reply:
as far as i could tell it works  and https://github.com/alpinejs/alpine/issues/900#issuecomment-1221111525